zipkin-web/src/main/scala/com/twitter/zipkin/web/Handlers.scala

```diff
@@ -178,7 +178,7 @@ class Handlers(queryExtractor: QueryExtractor) {
 
       val serviceDurations = groupedSpanTimestamps.map { case (n, sts) =>
         MustacheServiceDuration(n, sts.length, sts.map(_.duration).max / 1000)
-      }.toSeq
+      }.toSeq.sortWith((d1, d2) => d1.name.compareTo(d2.name) < 0)
```

Comment:
Sorting service durations alphabetically by name was more deterministic than the existing sort order (which seems random)